### PR TITLE
refactor: Lower camel case cleanup

### PR DESCRIPTION
**Pull request by mfederowicz**
### Summary

Cleanup some parts of code to meet one of the standards:

- lowerCamelCase names capitalize the first letter of each word, except the first which is always lowercase, even if it’s an acronym -  ([source](https://dart.dev/guides/language/effective-dart/style))

### CODE REVIEW

File: lib/Models/workout_data.dart [hunk 1]
```diff
@@ -72,13 +72,13 @@ class WorkoutData {
               newExerciseList.add(newexercise)
             });
       }
-      loaded_workout.exercises = newExerciseList;
+      loadedWorkout.exercises = newExerciseList;
       //workout = loaded_workout;
-    } catch (Exception) {
+    } catch (e) {
```
Comment (jorre127) [inline lib/Models/workout_data.dart]: Is there a reason you sometimes write exception and sometimes e? If not you should probably keep it consistent.

Comment (mfederowicz) [reply lib/Models/workout_data.dart]: ok i replaced catch(e) to catch(exception)

**Review by momshaddinury — APPROVED** [review]

**Comment by momshaddinury** [comment]
@tenshiAMD any idea why some of the files are showing every line changed even though we change a single line or some portion of it? Might be related to that editor config.

**Comment by mfederowicz** [comment]
@dinurymomshad afaik this problem is with changing endline encoding crlf(win) / cr(Macintosh ) / lf(Unix) - when i came to office iwill try to fix this - my dev env is all linux :)

**Comment by tenshiAMD** [comment]
> @tenshiAMD any idea why some of the files are showing every line changed even though we change a single line or some portion of it? Might be related to that editor config.

@dinurymomshad I think that is caused by the code editor, may I also know what code editor are you using?

also looks like the issues are in line endings, those files are using CRLF others are LF already.

**Review by tenshiAMD — CHANGES_REQUESTED** [review]
@mfederowicz hi please revert first the changes in lib/Models/exercise.dart, we should only see lines that are affected not the whole file that has been changed.

**Comment by mfederowicz** [comment]
@tenshiAMD maybe ill return to this topic after proper config editor, because Iam working in Android Studio on Linux, and after any changes in file with crlf it converts all file to selected format :(

**Comment by mfederowicz** [comment]
ok @tenshiAMD i reverted changes in lib/Models/exercise.dart

one small tip, problem is with .editorconfig configuration:

```
end_of_line = lf
```
it forces Android Studio to change end of line from orginal to LF - in case of lib/Models/exercise.dart it was CRLF, thats why when I commited only changes in few lines, github thinks that I changed all lines in selected file

**Comment by tenshiAMD** [comment]
@mfederowicz looks good to me however there are a lot of commits, can you do cleanups? first, rebase to alpha branch then commit into one. After that, I think it is good to merge then we address the line endings after this. Thanks a lot! 👍

**Comment by mfederowicz** [comment]
@tenshiAMD sorry for silly question but can you write me step by step ho to do those cleanups? I dont know what does it mean rebase to alpha branch especially in case forked repo :( of course I can make another clean branch from alpha and then apply changes from branch lowerCamelCase-cleanup :P

**Comment by tenshiAMD** [comment]
> @tenshiAMD sorry for silly question but can you write me step by step ho to do those cleanups? I dont know what does it mean rebase to alpha branch especially in case forked repo :( of course I can make another clean branch from alpha and then apply changes from branch lowerCamelCase-cleanup :P

@mfederowicz you might need to force push your branch though. I think it is much easier if you create another branch and close this PR then make a new one.

**Pull request closed without merging**